R/local.R

```diff
@@ -24,6 +24,12 @@ LocalCI <- R6Class(
     get_commit = function() {
       git2r::revparse_single(revision = "HEAD")$sha
     },
+    is_env = function(env, value) {
```

Comment:
Do we even need implementations in derived classes? I wonder if this should even be part of the `CI` class, maybe a simple exported function `has_env()` is enough?

Reply:
This is a general issue which would make all derived subclasses superfluous?